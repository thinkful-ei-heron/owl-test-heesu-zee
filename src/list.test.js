import React from 'react';
import ReactDOM from 'react-dom';
import List from './list';
import renderer from 'react-test-renderer';



describe('<List />', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<List name="balay" avatar="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1" inSession='true' onStage='true'/>, div);
        ReactDOM.unmountComponentAtNode(div);
    });
});

describe('<List />', () => {
    it('renders this UI as expected', () => {
        const tree = renderer.create(<List name="balay" avatar="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1" inSession='true' onStage='true'/>).toJSON();
        expect(tree).toMatchSnapshot(); 
    });
});