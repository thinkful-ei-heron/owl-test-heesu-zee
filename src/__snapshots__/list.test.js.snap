// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<List /> renders this UI as expected 1`] = `
<div
  className="people"
>
  <img
    alt="avatar"
    src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
  />
  <span>
    balay
  </span>
  <div />
  <span
    className="red-dot"
  />
  <p>
    
  </p>
</div>
`;
